Library/Homebrew/github_runner_matrix.rb

```diff
@@ -144,6 +144,8 @@ def generate_runners!
         ["#{version}#{ephemeral_suffix}", runner_timeout]
       end
 
+      # macOS 12-x86_64 is usually slower.
+      timeout += 30 if !(use_github_runner && github_runner_available) && macos_version <= :monterey
```

Comment:
```suggestion
      timeout += 30 if macos_version <= :monterey
```
Extra check doesn't really change anything TBH

Reply:
Not sure if I follow. If we're using the GitHub runner, then `timeout` is `GITHUB_ACTIONS_RUNNER_TIMEOUT`, which is already the longest timeout permitted by GitHub?

Reply:
Yes. It doesn't hurt to set that to a higher number; GitHub will just ignore it.